stonesoup/initiator/simple.py

```diff
@@ -165,7 +165,7 @@ class MultiMeasurementInitiator(GaussianInitiator):
     initiated only to then be removed shortly after.
     Does cause slight delay in initiation to tracker."""
 
-    prior_state: GaussianState = Property(doc="Prior state information")
+    prior_state: GaussianState = Property(default=None, doc="Prior state information")
```

Comment:
This change makes this a breaking change due to change in argument order, something ideally we'd look to avoid now at v1 release.

Reply:
I think the default value for ``prior_state`` could be omitted for this pull request. However I think it should be added in the future. Is there a plan on how to integrate breaking changes, maybe a development branch or v2 branch?